.github/workflows/run-tests.yml

```diff
@@ -116,6 +116,9 @@ jobs:
           /p:SkipPackageCheckForTemplatesTesting=true
           /bl:${{ github.workspace }}/artifacts/log/Debug/InstallSdkForTesting.binlog
 
+      - name: Install dotnet ef global tool
```

Comment:
It might be better to install this as part of the test? I've been doing some experiments with other .NET tools and found some success installing tools as --local.

Reply:
removed it for now. Will be added with #10138